### PR TITLE
WIP: Add interface for admin approval

### PR DESCRIPTION
**Pull request by abhishekkumar2718**

### CODE REVIEW

**Comment by abhishekkumar2718** [comment]
@apoorva-mk 

I am not clear over what else do we need for admin approval. Do suggest I have missed anything

File: lca_website/ngo/views.py [hunk 1]
```diff
@@ -32,4 +32,17 @@ def ngo_signup(request):
...
             return HttpResponseRedirect("/ngo/signup")
     else:
-        return render(request, "ngo_signup.html")
+        return render(request, "ngo/signup.html")
+
+def index(request):
+    template_data['ngos'] = Ngo.objects.filter(approved=False)
+    return render(request, 'ngo/index.html')
+
+def approve(request, ngo_id):
```
Comment (apoorva-mk) [inline lca_website/ngo/views.py:41]: Approving should come under the admin app that has already been initialised and we also need a login interface to the admin side.

Comment (apoorva-mk) [inline lca_website/ngo/views.py]: Can this come under the admin app that has already been initialised? 
Also, the admin interface login is needed.

Comment (abhishekkumar2718) [reply lca_website/ngo/views.py]: If we are using the admin app, there are no changes required. We would need to create a superuser on production:

Screenshot_2020-05-03 Change ngo Django site admin

Is that you meant?

Comment (apoorva-mk) [reply lca_website/ngo/views.py]: Yeah that suffices, the admin has only CRUD operations as of now in the plan, so this suffices.

Comment (apoorva-mk) [reply lca_website/ngo/views.py]: There are some other changes wrt to the tables, as we will also have details such as description and program details. So you may work on that.

**Pull request merged**